internal/consensus/byzantine_test.go

```diff
@@ -378,6 +372,7 @@ func TestByzantineConflictingProposalsWithPartition(t *testing.T) {
 		require.NoError(t, err)
 
 		conR := NewReactor(css[i], true) // so we don't start the consensus states
+		conR.conS.state.ConsensusParams.Blob.MaxBytes = types.MaxBlobSizeBytes
```

Comment:
This might not be needed as here by default the size is not 0, but it does not hurt.